grpc-proxy/README.md

```diff
@@ -0,0 +1,22 @@
+### Steps to run this sample:
+1) You need a Temporal service running. See details in README.md
+2) Run the following command to start the worker
+```
+go run worker/main.go
+```
+3) Run the following command to start the example
+```
+go run starter/main.go
```

Comment:
Does this not need the proxy to be running and therefore fail? Wouldn't we want the client and worker to connect to the proxy? I understand that we are showing a proxy created later, but I think it's clearer if we show that everything connects to that proxy instead of implementing data converters.

Reply:
Updated to use proxy for worker/starter.